app/src/components/explorer/Pagination.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div class="pagination-container">
+    <span>Page {{ cpage }} of {{ tpages }}</span>
+    <div class="pagination">
+      <button
+        :disabled="cpage <= 1"
+        @click.prevent="goToPage(1)"
+        class="pagination-button pagination-button-home viz-u-display__desktop"
+      >Home</button>
+      <button
+        :disabled="cpage <= 1"
+        @click.prevent="goToPage(cpage - 1)"
+        class="pagination-button pagination-button-prev viz-u-display__desktop"
+      >Prev</button>
+      <button
+        :disabled="cpage >= tpages"
+        @click.prevent="goToPage(cpage + 1)"
+        class="pagination-button pagination-button-next viz-u-display__desktop"
+      >Next</button>
+      <button
+        :disabled="cpage >= tpages"
+        @click.prevent="goToPage(tpages)"
+        class="pagination-button pagination-button-end viz-u-display__desktop"
+      >End</button>
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  name: 'pagination',
+  props: {
+    cpage: Number,
+    tpages: Number,
+  },
+  methods: {
+    goToPage (page) {
+      if (page != this.cpage) {
+        this.$emit('go-to-page', page)
+      }
+    },
+    pageExists (page) {
+      return page > 0 && page <= this.tpages
+    }
+  }
+}
+</script>
+<style lang="scss" scoped>
+@import "@/assets/css/abstract/_mixins.scss";
+@import "@/assets/css/abstract/_variables.scss";
+.pagination-container {
```

Comment:
I'm pretty sure these same stylings for pagination and pagination-container are also available as visualize-pagination and visualize-pagination-container in _visualize.scss